src/main/java/Jonathan.java

```diff
@@ -0,0 +1,110 @@
+import java.util.Objects;
+import java.util.Scanner;
+
+public class Jonathan {
+    public static void main(String[] args) {
+        Scanner input = new Scanner((System.in));
+        String command;
+        String line = "    ____________________________________________________________";
+        String welcome = line + "\n    Hello! I'm Jonathan\n" +
+                "    What can I do for you?\n" + line + "\n";
+        String goodbye = "    Bye. Hope to see you again soon!\n" + line;
+
+        Task[] list = new Task[100];
```

Comment:
It may be clearer to declare a constant (for example: TASK_LENGTH) to denote the length of array